tests/core/language_model/vllm/test_vllm_specific.py

```diff
@@ -13,13 +13,14 @@
 
 
 @pytest.fixture(scope="module")
-def chat_lm() -> VLLM:
+def chat_lm() -> Generator[VLLM, None, None]:
     llm = VLLM(
         model="sbintuitions/tiny-lm-chat",
         model_kwargs={
             "seed": 42,
             "gpu_memory_utilization": 0.1,
             "enforce_eager": True,
+            "dtype": "float32",
```

Comment:
This is needed to pass the test: `test_batch_compute_log_probs_approximates_hf_lm`.
In this test case, Huggingface LM uses float32; so float32 should be a reasonable choice for VLLM, too.